macos/FluentUI/Button/Button.swift

```diff
@@ -211,6 +211,13 @@ open class Button: NSButton {
 		return true
 	}
 
+	open override func drawFocusRingMask() {
+		// Ensure we draw the focus ring around the entire button bounds
+		// rather than just around the image or title.
+		let path = NSBezierPath(roundedRect: bounds, xRadius: cornerRadius, yRadius: cornerRadius)
+		path.fill()
```

Comment:
why is it fill rather than stroke?

Reply:
This is the mask for the focus ring.  I'm not drawing the ring itself.

Reply:
i.e. how it works is that the actual focus ring hugs the outside edge of this shape.